backend/routes/usernameRoute.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+
+const users = {
+    "john_doe": { id: 1, username: "john_doe", email: "john@example.com" },
+    "jane_smith": { id: 2, username: "jane_smith", email: "jane@example.com" },
+    // Add more users as needed
+};
```

Comment:
it's fine if you're doing this to test some user CRUD operations, but this shouldn't be merged into the codebase